demos/tests/run_tests.py

```diff
@@ -17,8 +17,12 @@
 """
 Test script for the demos.
 
-For the tests to work, the test data directory must contain a "ILSVRC2012_img_val"
-subdirectory with the ILSVRC2012 dataset.
+For the tests to work, the test data directory must contain:
+* a "BraTS" subdirectory with BraTS 2017 dataset in NIFTI format (see http://medicaldecathlon.com,
```

Comment:
I thought `medicaldecathlon` for brain tumor and `BraTS 2017` are different data sets. Are you sure they are the same?

Reply:
done